lib/rubycritic/generators/json/simple.rb

```diff
@@ -24,6 +25,18 @@ def data
             score: @analysed_modules.score
           }
         end
+
+        def file_directory
+          @file_directory ||= Pathname.new(Config.root)
+        end
+
+        def file_name
+          'report.json'
+        end
+
+        def file_pathname
+          File.join(file_directory, file_name)
```

Comment:
Why use File here and not Pathname? Also I believe you can merge `file_name` and `file_pathname`, this can be one method.